backends/pkg/eventmesh/v1/builder.go

```diff
@@ -90,6 +137,21 @@ func BuildEventMesh(ctx context.Context, clientset versioned.Interface, dynamicC
 		}
 	}
 
+	subscriptions, err := clientset.MessagingV1().Subscriptions(metav1.NamespaceAll).List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		logger.Errorw("Error listing subscriptions", "error", err)
+		return EventMesh{}, err
+	}
+
+	for _, subscription := range subscriptions.Items {
+		err := processSubscription(ctx, &subscription, subscribableMap, etByNamespacedName, dynamicClient, logger)
+		if err != nil {
+			logger.Errorw("Error processing subscription", "error", err)
+			// do not stop the Backstage plugin from rendering the rest of the data, e.g. because
+			// there are no permissions to get a single subscriber resource
+		}
```

Comment:
Shouldn't this skip be associated with the specific error `apierrors.IsUnauthorized(err)` and fail for others ?